tests/functional/test_download.py

```diff
@@ -139,7 +139,7 @@ def test_download_vcs_link(script):
     It should allow -d flag for vcs links, regression test for issue #798.
     """
     result = script.pip(
-        'install', '-d', '.', 'git+git://github.com/pypa/pip-test-package.git'
+        'download', '-d', '.', 'git+git://github.com/pypa/pip-test-package.git'
```

Comment:
And the more I think about it, the more I think we should.
New features might be added to `pip download` without affecting the current `pip install --download` behavior and having tests will help ensure we do not break anything :)